archive/f/f-star/Baklava.fst

```diff
@@ -0,0 +1,17 @@
+module Baklava
+
+open FStar.IO
+open FStar.Math.Lib
+open FStar.Mul
+
+let baklava_line (n:nat {n <= 20}) : string =
+  let num_spaces:nat = (abs (n - 10)) in
+  let num_stars:nat = 21 - 2 * num_spaces in
+  (String.make num_spaces ' ') ^ (String.make num_stars '*') ^ "\n"
```

Comment:
What is happening here? I assume a highlighting issue with GitHub interpreting this as the wrong language.

Reply:
Not really sure. GitHub knows about the `F*` language. See [this repo](https://github.com/amiller/fstar374). Maybe there's a bug in the syntax highlight for that language.